components/raftstore/src/store/fsm/peer.rs

```diff
@@ -4545,6 +4551,23 @@ where
         self.register_pd_heartbeat_tick();
     }
 
+    fn on_refresh_region_buckets(
+        &mut self,
+        region_epoch: metapb::RegionEpoch,
+        region_buckets: metapb::Buckets,
+    ) {
+        let region = self.fsm.peer.region();
+        if util::is_epoch_stale(&region_epoch, region.get_region_epoch()) {
+            warn!(
```

Comment:
`info` should be enough as split/merge is expected behaviors.

Reply:
done